### ISSUE TITLE
blog should allow one to see older blog posts

### ISSUE THREAD

**Issue by adf-ncgr** [LegumeFederation/legfed-django-site#35]
unless there's something obvious I'm missing, it seems there's no way to access the older blog posts without being an admin. not ideal; @svengato maybe you could have a look at this sometime; not urgent but hopefully something that can be fixed relatively easily

**Comment by svengato** [comment]
> unless there's something obvious I'm missing, it seems there's no way to access the older blog posts without being an admin. not ideal; @svengato maybe you could have a look at this sometime; not urgent but hopefully something that can be fixed relatively easily

I added an Archive plugin that partially solves the problem. The year links work as expected, but the month links are incorrect due to an off-by-one error that I will look into tomorrow. Let me know what you think,
  http://dev.lis.ncgr.org:50003/en/

-Sven-

**Comment by adf-ncgr** [comment]
looks good- I imagine it would be relatively easy to put that on its own page and just link to it from beneath the "current" post(s)? this would (in future) prevent the front page from getting too crowded and (in present) prevent me from getting depressed by how few posts we have...

**Comment by svengato** [comment]
The month-off-by-one error mysteriously disappeared - did you fix anything?

**Comment by svengato** [comment]
The blog archive now has its own page, accessed from a link on the home page.

**Comment by adf-ncgr** [comment]
no, unless it was by the mere act of observing. Schroedinger's Gato?
